Assets/Scripts/Game/PlayerActivate.cs

```diff
@@ -470,7 +470,7 @@ void Update()
             Transform doorOwner)
         {
             StaticDoor door;
-            if (doors.HasHit(hit.point, out door) || CustomDoor.HasHit(hit, out door))
+            if (CustomDoor.HasHit(hit, out door) || doors.HasHit(hit.point, out door))
```

Comment:
Is this for when you have a classic door and a custom door with a partial overlap? I'm not necessarily against it, but i'm not sure this should happen in the first place. If this actually solves an issue you found i'm happy to merge :)

Reply:
That is correct. If there is an overlap between the two, I want CustomDoors to have priority. 

My other comment explains how this solves an issue I found.